lib/orderbook/OrderBook.ts

```diff
@@ -816,6 +816,23 @@ class OrderBook extends EventEmitter {
     return order;
   }
 
+  public removeOwnOrders = () => {
+    const successfullyRemovedOrderIds = [];
+    const failedToRemovedOrderIds = [];
+
+    for (const localId of Array.from(this.localIdMap.keys())) {
+      try {
+        this.logger.debug(`Trying to remove order with id: ${localId}`);
```

Comment:
I think this logging statement should be removed, or at most be set to `trace`, since there are other logging statements when the orders are actually removed.